userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -213,7 +214,33 @@ site's [configuration file][].
 [configuration file]: https://gohugo.io/getting-started/configuration/#configuration-file
 [primary color]: #site-colors
 
+### Styling your brand logo and name
 
+The default Docsy navbar (`.td-navbar`) contains your navbar-brand, consisting
```

Comment:
"contains your navbar-brand" -> "identifies your overall project"

Reply:
Or maybe "The default Docsy navbar displays your site identity, consisting of..." Better?

Reply:
Done